vent/service/consumer.go

```diff
@@ -217,11 +217,20 @@ func (c *Consumer) Run(parser *sqlsol.Parser, abiSpec *abi.AbiSpec, stream bool)
 								// unpack, decode & build event data
 								eventData, err := buildEventData(spec, parser, event, abiSpec, c.Log)
```

Comment:
My view it is better to monitor errors rather than failing on an event decode. I see more problems with the current failure mode than I see advantages. Particularly on extended systems where an operator of vent does not have full control over the network and therefore lacks the ability to control events raised by any smart contracts.